rust/src/compose.rs

```diff
@@ -250,12 +253,15 @@ pub(crate) fn compose_image(args: Vec<String>) -> CxxResult<()> {
 
     let label_args = opt.labels.into_iter().map(|v| format!("--label={v}"));
 
+    //Provide --prior-build optionally to follow prior packing structure (integrate to workflow)
```

Comment:
In `compose image` we need to optionally supply `--prior-build` from `cosa` to `container-encapsulate`. This becomes relevant when we need to use to use the same packing structure as the previous build. @cgwalters how could we accomplish this?

Reply:
There's code in coreos-assembler today to use previous builds.  The production pipelines use `cosa buildfetch`.

When we do that, we can fetch the previous `.ociarchive` from s3 and use it.